addOns/quickstart/src/main/java/org/zaproxy/zap/extension/quickstart/launch/ExtensionQuickStartLaunch.java

```diff
@@ -144,7 +145,11 @@ private JButton getLaunchToolbarButton() {
             launchToolbarButton = new JButton();
             launchToolbarButton.setToolTipText(
                     Constant.messages.getString("quickstart.toolbar.button.tooltip.launch"));
-            launchToolbarButton.addActionListener(e -> launchPanel.launchBrowser());
+            launchToolbarButton.addActionListener(
+                    e -> {
+                        launchPanel.launchBrowser();
+                        Stats.incCounter("stats.ui.toolbar.quickstart.browserlaunch");
```

Comment:
Why no button on this one and why the quickstart prefix? Probably worth standardize something at some point.